### PR TITLE
Add say exercise

### PR DESCRIPTION
**Pull request by tgecho**
I'm not totally happy with the clarity of the example for this one, but it all seems to work. We'll see what everyone else does with it!

@parkerl

### CODE REVIEW

File: exercises/say/Say.example [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    if number < 0 then
+        Err Negative
+    else if number < 20 then
+        Ok (saySmall number)
+    else if number < 100 then
+        if number % 10 == 0 then
+            Ok (saySmall number)
+        else
+            Ok (sayTens (number // 10 * 10) ++ "-" ++ (saySmall (number % 10)))
+    else if number < k then
```
Comment (lukewestby) [inline exercises/say/Say.example]: i think it would be okay to hardcode each of these constants

**Comment by lukewestby** [comment]
Looks great!

File: exercises/say/Say.example (hunk 1 above)
Comment (tgecho) [inline exercises/say/Say.example]: The `k`, `t`, etc...? I also thought about expanding them to at least full words.

Comment (lukewestby) [inline exercises/say/Say.example]: full words would be good too. it took me a second to figure out that they were abbreviations for the SI prefixes

**Pull request merged**